pkg/operator/controllers/machinehealthcheck/machinehealthcheck_controller.go

```diff
@@ -31,6 +32,8 @@ type Reconciler struct {
 }
 
 func NewReconciler(arocli aroclient.Interface, dh dynamichelper.Interface) *Reconciler {
+	// registers types such as PrometheusRule
+	monitoringv1.AddToScheme(scheme.Scheme)
```

Comment:
hmm. I'd have to do some more digging but i'm wondering if this should go into `pkg/util/scheme/scheme.go` wdyt?

Reply:
That sounds like a good place for it, I will look into it and make the changes if appropriate.